Framework/Algorithms/src/CorelliCrossCorrelate.cpp

```diff
@@ -131,8 +131,8 @@ void CorelliCrossCorrelate::exec() {
 
   // Calculate the duty cycle and the event weights from the duty cycle.
   double dutyCycle = totalOpen / sequence.back();
-  float weightTransparent = static_cast<float>(1.0 / dutyCycle);
-  float weightAbsorbing = static_cast<float>(-1.0 / (1.0 - dutyCycle));
+  float weightTransparent = 1.0;
```

Comment:
For all the times you use weightTransparent you could use a string with value "42", and it would not change anything. You just use it on line 137 to print it.